.github/workflows/test.yml

```
@@ -0,0 +1,217 @@
+        uses: ./.github/actions/upload-test-results
+        with:
+          name: test-results
+          path: test-results/
+          retention-days: 7
+          status: ${{ job.status }}
+          token: ${{ secrets.CODECOV_TOKEN }}
+          codecov-files: test-results/junit.xml
+          coverage-report: test-results/coverage.xml
+
+  test-behave:
+    runs-on: ${{ matrix.os }}
```

actionlint (1.7.4)

property "os" is not defined in object type {}

(expression)